_dev/offboarding.md

```diff
@@ -48,11 +48,11 @@ all packages using `brew upgrade`.
 MySQL runs as a background service. You must stop the background service
 before removing it.
 
-``` console
+{% highlight console linenos %}
 $ brew services stop mysql@5.7
 $ brew uninstall --force mysql@5.7
```

Comment:
This is out of date. We're use the default MySQL/MariaDB now, so no need to include `@5.7`.